### DISCUSSION TITLE
Gratitude for Your Incredible AI Project

### DISCUSSION THREAD

**Discussion by sambickeita** [AI4Finance-Foundation/FinGPT#4] [General]
I  express my gratitude for making this AI project available to everyone. It's an outstanding contribution that benefits us all

**Comment by YangletLiu** [comment]
> I express my gratitude for making this AI project available to everyone. It's an outstanding contribution that benefits us all

Thanks for your kind words.  We will continue update frequently each week, and hopefully releasing pre-trained models regularly.
